readthedocs/extra/basic/entities.rst

```diff
@@ -61,16 +62,18 @@ Entities vs. Input Entities
 
     Don't worry if you don't understand this section, just remember some
     of the details listed here are important. When you're calling a method,
-    don't call ``.get_entity()`` before, just use the username or phone,
+    don't call ``.get_entity()`` beforehand, just use the username or phone,
     or the entity retrieved by other means like ``.get_dialogs()``.
 
 
-To save bandwidth, the API also makes use of their "input" versions.
+On top of the normal types, the API also make use of what they call their
+``Input*`` versions of objects.
 The input version of an entity (e.g. ``InputPeerUser``, ``InputChat``,
-etc.) only contains the minimum required information that's required
-for Telegram to be able to identify who you're referring to: their ID
-and hash. This ID/hash pair is unique per user, so if you use the pair
-given by another user **or bot** it will **not** work.
+etc.) only contains the minimum information that's required from Telegram
+to be able to identify who you're referring to:
+a Peer's **ID** and **hash**.
```

Comment:
You could: do <code>\`\`Peer\`\`'s</code> to further emphasize they're actual types.